app/server.ts

```diff
@@ -19,7 +19,7 @@ const app = next({ dev });
 const handle = app.getRequestHandler();
 
 const { json, urlencoded } = bodyParser;
-
+const bodyParserLimit = '50mb';
```

Comment:
where does 50mb come from? 
Can we calculate the maximum expected payload size, assuming 32k characters for each of those inputs that don't have a max length?

Reply:
50mb comes from various examples around the web as well as it is the limit that is set in cif.

https://github.com/bcgov/cas-cif/blob/develop/app/server/index.ts

Reply:
Given our limited storage sizes, we might want to lower that to prevent users from filling the database by forging a 50Mb JSON. 5Mb might be more than enough?

@wenzowski, any recommendation here?

Reply:
What error do we get when we exceed the `bodyParserLimit`? How can we monitor for users hitting the parse limit in the normal course of system operation?

Reply:
We get a 413 error returned back from the server.

Reply:
@marcellmueller, can you change this to 5Mb? We should be able to monitor for those errors with sysdig